docs/contributing.rst

```diff
@@ -0,0 +1,67 @@
+
+Anitya Development Guide
+========================
+
+Anitya welcomes contributions! Our issue tracker is located on
+`GitHub <https://github.com/fedora-infra/anitya/issues>`_.
+
+
+Contribution Guidelines
+-----------------------
+
+When you make a pull request, a Fedora Infrastructure team member will review your
+code. Please make sure you follow the guidelines below:
+
+Code style
+^^^^^^^^^^
+
+We follow the `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ style guide for Python.
+The test suite includes a test that enforces the required style, so all you need to do is
+run the tests to ensure your code follows the style. If the unit test passes, you are
+good to go!
+
+Unit tests
+^^^^^^^^^^
+
+All unit tests must pass. All new code should have 100% test coverage.
+Any bugfix should be accompanied by one or more unit tests to demonstrate the fix.
+If you are unsure how to write unit tests for your code, we will be happy to help
+you during the code review process.
+
+You can run the unit tests by running ``python setup.py test`` in the root of the
+repository.
+
+
+Development environment
+-----------------------
+
+The development environment is set up using `Ansible <https://www.ansible.com/>`_. You can use
```

Comment:
![You make me sad](http://2.bp.blogspot.com/-FjA7XRDc_pc/Ur_8o46iEqI/AAAAAAAAPlE/Gmpo52_kH30/s1600/you-make-me-sad.gif)